generate_dlc_image_release_information.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+import logging
+import json
+import os
+import sys
+import shutil
+import tarfile
+
+import boto3
+
+from botocore.exceptions import ClientError
+from release.dlc_release_information import DLCReleaseInformation
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+LOGGER.setLevel(logging.INFO)
+
+
+def write_to_file(file_name, content):
+    """
+    Uses json package to dump content to a file. The content may or may-not be json-structured.
+    :param file_name:
+    :param content:
+    :return:
+    """
+    with open(file_name, "w") as fp:
+        fp.write(content)
+        LOGGER.info(f"Content written to file: {file_name}")
+
+
+def upload_to_S3(local_file_path, bucket_name, bucket_key):
+    """
+    Upload a local file to s3 with specified bucket name and key
+    :param local_file_path: string
+    :param bucket_name: string
+    :param bucket_key: string
+    :return:
+    """
+    _s3 = boto3.client('s3')
+    try:
+        _s3.upload_file(local_file_path, bucket_name, bucket_key)
+    except ClientError as e:
+        LOGGER.error("Error: Cannot upload file to s3 bucket.")
+        LOGGER.error("Exception: {}".format(e))
+        raise
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Specify bucket name to upload release information for DLC")
+    parser.add_argument("--artifact-bucket", type=str, required=True)
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+
+    args = parse_args()
+    dlc_artifact_bucket = args.artifact_bucket
+
+    dlc_account_id = os.getenv("TARGET_ACCOUNT_ID_CLASSIC")
+    dlc_tag = os.getenv("TAG_WITH_DLC_VERSION")
+    dlc_repository = os.getenv("TARGET_ECR_REPOSITORY")
+    dlc_region = os.getenv("REGION")
```

Comment:
Can we ensure that these env variables actually have values correctly configured, and that if they don't, then this workflow quits without generating any release information?